<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二维码识别QT上位机</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>172</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>端口：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit_2">
       <property name="placeholderText">
        <string>5000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>172</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>主机：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostLineEdit_2">
       <property name="placeholderText">
        <string>10.10.10.103</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>41</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Connect to server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Disconnect from server</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>484</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_status">
       <property name="maximumSize">
        <size>
         <width>9999</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>551</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send this:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>二维码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Update">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QRCode" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>230</width>
      <height>210</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <zorder>textEdit_log</zorder>
   <zorder>pushButton_send</zorder>
   <zorder>lineEdit_message</zorder>
   <zorder>splitter</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>pushButton_connect</zorder>
   <zorder>pushButton_disconnect</zorder>
   <zorder>widget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QRCode</class>
   <extends>QWidget</extends>
   <header>qrcode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
